app/experimenter/experiments/models.py

```diff
@@ -668,6 +668,28 @@ def format_firefox_versions(self):
         else:
             return self.firefox_min_version
 
+    @property
+    def versions_integer_list(self):
+        min_integer = int(
```

Comment:
I feel like we do these regex extractions in a bunch of places, would it make sense to move this to a separate model method?  This method here shouldn't be as long as it is....

Reply:
@jaredkerim yes I was debating making them their own model method. So I will!